content/02-types.md

```diff
@@ -234,11 +234,11 @@ The above code will yield an instance of class `Point`, which is assigned to a v
 <!--label:types-class-inheritance-->
 #### Inheritance
 
-Classes may inherit from other classes, which in Haxe is denoted by the `extends` keyword:
+Classes may inherit from other classes, and is denoted by the `extends` keyword:
```

Comment:
The changed sentence doesn't make sense. Maybe `… classes; this is denoted …` ?

Reply:
Agreed, this sentence ended up sounding weird, I'll rephrase